docs/content/python/getting-started/quickstart/index.md

```diff
@@ -51,10 +52,10 @@ Oso needs three pieces of information to make an authorization decision:
 - What are they trying to do? (the "action")
 - What are they doing it to? (the "resource")
 
-You'll pass these pieces of information are to Oso's `{{% exampleGet "isAllowed" %}}` method: `{{% exampleGet "isAllowed" %}}(actor, action, resource)`.
+You'll pass these pieces of information into to Oso's `{{% exampleGet "isAllowed" %}}` method: `{{% exampleGet "isAllowed" %}}(actor, action, resource)`.
 `{{% exampleGet "isAllowed" %}}` will return `True` or `False`, and your application can choose how to enforce that decision.
 
-Here's a program that only allows access to a page if the current user has a role that is allowed to read it.
+Here's a program that only allows access to a page if the current user has a role that is allowed to read it. (More precisely, this is a program that allows access if the policy allows it, and the policy allows access to users with the correct role — note the distinction between "policy" and "program".)
```

Comment:
I don't quite follow the "distinction" piece here. You're trying to emphasize that the policy is general while the specific policy written here only allows access to users with the right role?

Reply:
I may have misunderstood your comment above:

```
This isn't strictly correct.

The program only allows access to a page if the policy allows the current user to perform the read action on it.

Since the policy uses roles, the user must have a role that's allowed to read it.

How important do you think this distinction is?
```

I thought your point was: there is a difference between a policy (which has rules) and a program (which executes). Is that what you meant? And if so, does this parenthetical clear it up?

Reply:
My point was just the Python code here isn't doing anything specific to roles. It's calling the policy, which allows a user if they have the right role.

We might just say something like:

> Here’s a program that allows access to a page if Oso allows the current user to read it.

(The reason I didn't include "policy" here is we don't introduce that concept until the paragraph after the code snippet.et